src/main/java/step2/LotteryNumbers.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LotteryNumbers {
+    private List<Integer> lottoNumberList;
+
+    public LotteryNumbers() {
+        lottoNumberList = new ArrayList<>();
+    }
+
+    public void addLottoNumber(int number) {
+        lottoNumberList.add(number);
+    }
+
+    public List<Integer> getLottoNumberList() {
+        return lottoNumberList;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder("[");
+        for (Integer number : lottoNumberList) {
+            stringBuilder.append(number).append(",");
+        }
+
+        int lastCommaIndex = stringBuilder.lastIndexOf(",");
+        stringBuilder.deleteCharAt(lastCommaIndex);
+
+        return stringBuilder.toString();
```

Comment:
prefix인 "[", 일종의 delimiter인 "," suffix인 "]"로 String을 묶고 싶다면, Stream을 사용해서 Collectors.joining()을 사용하는건 어떨까요?

그리고 List.toString()이 원하시는 포맷과 비슷하게 String을 만들어주긴합니다 :) 한번 확인해보세요!
```suggestion
   return lottoNumberList.toString();
```